e2e/test/scenarios/dashboard/dashboard.cy.spec.js

```diff
@@ -352,6 +352,40 @@ describe("scenarios > dashboard", () => {
           .and("contain", "18,760");
       });
 
+      it("should save changes to a dashboard after using the 'Add a chart' button from an empty tab (metabase#53132)", () => {
+        cy.log("add an existing card");
+        cy.findAllByTestId("dashboard-header").icon("pencil").click();
+        cy.findAllByTestId("dashboard-header").icon("add").click();
+        H.sidebar().findByText("Orders, Count").click();
+        cy.findAllByTestId("dashboard-header").icon("add").click();
+
+        cy.log("create a tab to access emtpy state again");
+        H.createNewTab();
+        cy.findByTestId("dashboard-empty-state")
+          .findByText("Add a chart")
+          .click();
+
+        cy.log("save changes before leaving");
+        H.sidebar().findByText("New SQL query").click();
+        H.modal().findByRole("button", { name: "Save changes" }).click();
+
+        cy.log("create a dashboard question");
+        H.NativeEditor.focus().type("SELECT 1");
+        H.queryBuilderHeader().findByText("Save").click();
+        H.modal().within(() => {
+          cy.findByPlaceholderText(/name of your question/i).type(
+            "Foo question",
+          );
+          cy.findByRole("button", { name: "Save" }).click();
+        });
```

Comment:
There is a handy helper that's doing all this for you. It's called `saveQuestion`.